hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/tools/TestDFSZKFailoverController.java

```diff
@@ -226,9 +228,11 @@ public void testWithBindAddressSet() throws Exception {
     DFSZKFailoverController zkfc = DFSZKFailoverController.create(
         conf);
     String addr = zkfc.getRpcAddressToBindTo().getHostString();
+    String nameserviceId = conf.get(DFS_NAMESERVICE_ID);
 
     assertEquals("Bind address " + addr + " is not wildcard.",
         addr, WILDCARD_ADDRESS);
+    assertEquals(nameserviceId, NAMESERVICE_ID);
```

Comment:
nit: `assertEquals(expected, actual)`.

Reply:
Is this usage wrong?

Reply:
Yes: `assertEquals(NAMESERVICE_ID, nameserviceId)`